### DISCUSSION TITLE
Corundum on Intel Agilex-7 I-series Dev Kit

### DISCUSSION THREAD

**Discussion by HxJi** [corundum/corundum#200] [General]
Hi,

Just wonder if there are any plans to make Corundum run on the recent Intel Agilex-7 I-series FPGA since it is more powerful with new features.

**Comment by alexforencich** [comment]
Yes, but likely only for gen 3 or gen 4 x16 as gen 5 x16 may be problematic from a timing perspective.

I'm also considering doing a rewrite in SV that might also be able to support faster link rates, including gen 5 x16 and 400G Ethernet. But that's more of a long-term thing.
